com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDSubShaderUtilities.cs

```diff
@@ -218,6 +218,16 @@ public static void AddDoubleSidedProperty(PropertyCollector collector, DoubleSid
                 hlslDeclarationOverride = HLSLDeclaration.UnityPerMaterial,
                 value = new Vector4(1, 1, -1, 0)
             });
+            collector.AddShaderProperty(new Vector1ShaderProperty
+            {
+                enumNames = { "MatchMaterial", "ForcesOn", "ForcesOff" }, // values will be 0, 1 and 2
```

Comment:
Please rename enum values in the UI to:
`MatchMaterial` to `Auto`
`ForcesOn` to `On`
`ForcesOff` to `Off`